api/spec/requests/api/v1/contacts_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "swagger_helper"
+
+RSpec.describe "Contacts", type: :request do
+  path "/contacts" do
+    post "Sends an email to the provided address" do
+      consumes "application/json"
+      tags "Contacts"
+      parameter name: :body, in: :body, schema: ApiDocumentation::DryTypesParser.convert(
+        Types::Hash.schema(
+          data: Types::Hash.schema(
+            attributes: Types::Hash.schema(
+              email: Types::String,
+              full_name: Types::String,
+              message: Types::String
+            )
+          )
+        )
+      )
+
+      response "204", "Sent the email" do
+        let(:body) do
+          {
+          	data: {
+          		attributes: {
+          			email: "connor@castironcoding.com",
```

Comment:
We don't really want to commit your actual email address and name to the repository, since presumably this will appear in our documentation. Use more appropriate fake data: (email@email.com, John Smith, etc.)

Reply:
Theses are just values that get passed into the rswag test so it can run it against the rails app, but that still makes sense that there is better test data to use. Thanks for keeping an eye out for that.

I'll go with Max's default name of "John Rambo" then 😄